editor/src/components/canvas/canvas-strategies/strategies/absolute-resize-bounding-box-strategy.spec.browser2.tsx

```diff
@@ -212,6 +212,49 @@ export var ${BakedInStoryboardVariableName} = (props) => {
 }
 `
 
+const projectWithAspectRatioLockedElement = `import * as React from 'react'
+import { Scene, Storyboard } from 'utopia-api'
+
+export var storyboard = (
+  <Storyboard data-uid='0cd'>
+    <div
+      data-aspect-ratio-locked
+      style={{
+        backgroundColor: '#0091FFAA',
+        position: 'absolute',
+        left: 33,
+        top: 307,
+        width: 363,
+        height: 426,
```

Comment:
I think it'd make the readability better if these numbers would be round, like 300x40

Reply:
done!